src/models/disassemblyoutput.cpp

```diff
@@ -213,6 +232,14 @@ DisassemblyOutput::ObjectdumpOutput DisassemblyOutput::objdumpParse(const QByteA
 
         const auto parts = asmLine.split(QLatin1Char('\t'));
 
+        if (parts.size() == 1 && asmLine.endsWith(QLatin1Char(':'))) {
```

Comment:
it's unfortunate that you do so much in this patch - I would really appreciate if you could split it up - one patch that just ports away from the old QTextDocument usage to the new highlighter, then another patch for adding the custom highlighting

Reply:
this wasn't done yet?